testsuite/integration-tests-spring/pom.xml

```diff
@@ -117,6 +116,13 @@
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-json-binding-provider</artifactId>
             <scope>provided</scope>
+            <exclusions>
+                <!-- todo: this should be fixed in RESTEasy -->
+                <exclusion>
+                    <groupId>org.eclipse.parsson</groupId>
+                    <artifactId>parsson</artifactId>
+                </exclusion>
```

Comment:
I'm not sure I understand why this would be needed? RESTEasy should be exporting this. It can be excluded, but it won't work without an implementation.